librz/lang/lang.c

```diff
@@ -46,11 +45,16 @@ RZ_API RzLang *rz_lang_new(void) {
 
 RZ_API void rz_lang_free(RzLang *lang) {
 	if (lang) {
```

Comment:
Could you please flip the condition here while you are on it?

Reply:
Also add `RZ_NULLABLE`

Reply:
I don't think we put NULLABLE in all _free methods. I *think* we have a convention that _free will always accept NULL and do nothing.